src/pages/home/HeaderView.js

```diff
@@ -273,6 +274,15 @@ function HeaderView(props) {
                                 <ThreeDotsMenu
                                     anchorPosition={styles.threeDotsPopoverOffset(props.windowWidth)}
                                     menuItems={threeDotMenuItems}
+                                    shouldSetModalVisibility={false}
+                                    onIconPress={() => {
+                                        const {composerRef, editComposerRef, isFocused, isEditFocused} = ReportActionComposeFocusManager;
+                                        if (isFocused()) {
+                                            composerRef.current.blur();
+                                        } else if (isEditFocused()) {
+                                            editComposerRef.current.blur();
+                                        }
+                                    }}
```

Comment:
@ginsuma This change causes regression. When closing the modal, the composer doesn't re-focus in case when it was focused before. The issue is that when the composer was not focused, after opening and closing the modal, the composer shouldn't focus.

Reply:
@pecanoro any thoughts? I think that the composer should be refocused when it was focused before, like on the emoji picker